src/main/java/com/backoffice/upjuyanolja/domain/member/controller/OwnerAuthController.java

```diff
@@ -60,12 +61,12 @@ public ResponseEntity<SuccessResponse<OwnerSignupResponse>> ownerSingup(
     }
 
     @PostMapping("owners/signin")
-    public ResponseEntity<SuccessResponse<SignInResponse>> ownerSingup(
+    public ResponseEntity<SuccessResponse<SignInResponse>> ownerSignup(
```

Comment:
signin으로 수정해야 할 것 같아요~~😊
```suggestion
    public ResponseEntity<SuccessResponse<SignInResponse>> ownerSignin(
```